react-carousel-component/src/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import Carousel from './carousel';
+
+const pokemon = {
+  bulbasaur: 'images/001.png',
+  charmander: 'images/004.png',
+  pikachu: 'images/007.png',
+  squirtle: 'images/025.png',
+  jigglypuff: 'images/039.png'
+};
```

Comment:
I recommend using an array here instead of an object. If you want, you can add a `name` property to each object in the array

```js
const pokemon = [
  { name: 'bulbasaur', src: 'images/001.png' },
  // ...
]
```